### ISSUE TITLE
Pixel Perfect in ScummVM

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoScummVM#289]
The latest release of ScummVM I've tested (2.3.0git15811, Apr 28 2021) allows the command "--stretch-mode=pixel-perfect" to enable pixel perfect. ScummVM will increase or decrease the scale of the game in accordance to the pixel perfect algorithm as you manually drag the window size larger or smaller. This is great but obviously being able to specify the window size at launch would be best for our use case. 

This is also meant to be integrated into this release with "--window-size=W,H" which apparently only works in conjunction with "--gfx-mode=opengl". This is all fine, however in testing I noticed that the Scummvm.ini also records the last window size you had, and once its does so the setting in the ini seems to override anything you set at the command line.  I could in theory inject a window height and window width setting of my own into the users scummvm.ini but that is not an option i really wish to pursue.

I will see what happens with the next official stable build before proceeding further with pixel perfect implementation.

**Comment by exoscoriae** [comment]
I agree. It seems experimental and new. Any work we put into supporting it today is likely to be undone by the next scummvm release.

Seems like a nice thing to hold off on for a future update.

**Comment by Python-Exoproject** [comment]
--window-size has been confirmed to be not working in scummvm from the command line and reported to the developers. In the meantime we have setup a workaround that uses the settings in the ini.
